### PR TITLE
fix saving energy offset and add global save

### PR DESCRIPTION
**Pull request by steinnymir**
This PR adds the method to save energy offset parameters, and a method to call all save parameter methods in SedProcessor.

### CODE REVIEW

File: sed/calibrator/energy.py [hunk 1]
```diff
@@ -1523,16 +1524,24 @@ def add_offsets(
                         signs.append(v["sign"])
                     except KeyError as exc:
                         raise KeyError(f"Missing sign for offset column {k} in config.") from exc
-                    preserve_mean.append(v.get("preserve_mean", False))
-                    reductions.append(v.get("reduction", None))
+                    pm = v.get("preserve_mean", False)
+                    if pm == "false":
+                        pm = False
+                    elif pm == "true":
+                        pm = True
```
Comment (rettigl) [inline sed/calibrator/energy.py:1531]: Shouldn't the yaml library handle this correctly? Off course, these should not be strings in the yaml...

File: sed/calibrator/energy.py [hunk 2]
```diff
@@ -1523,16 +1524,24 @@ def add_offsets(
...
+                    preserve_mean.append(pm)
+                    rd = v.get("reduction", None)
+                    if rd == "none":
```
Comment (rettigl) [inline sed/calibrator/energy.py:1534]: maybe add .lower() ?

File: sed/calibrator/energy.py (hunk 1 above)
Comment (rettigl) [reply sed/calibrator/energy.py:1531]: otherwhise, also add .lower() ?

**Review by rettigl — APPROVED** [review]
LGTM

File: sed/calibrator/energy.py (hunk 1 above)
Comment (steinnymir) [reply sed/calibrator/energy.py:1531]: no, it does not automatically pick up bools and None

**Pull request merged**